vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -870,8 +870,10 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
     override fun onResume() {
         super.onResume()
 
-        // search the loading view from the upper view
-        mLoadingView = view!!.findViewById(R.id.vector_settings_spinner_views)
+        // search the loading view from the upper
+        // find the view from parent activity
+        mLoadingView = activity!!.findViewById(R.id.vector_settings_spinner_views)
```

Comment:
Note that I really don't like the fact that a Fragment manipulates View from its Activity, but the code is full of dirty practices like that, so it is out of scope of this PR...

Reply:
So should I move the loading view to fragment only. ?

Reply:
No, You won't be able to do that because it's a pref fragment. I was more thinking of a way for Fragment to ask its Activity to show/hide the loading View.

Reply:
We can have Interface Call back. VectorSettingPreferencesFragment Can have interface for displayLoadingView and HideLoadingView(). actvity can give the implementation at the time of initialization of this fragment.

Reply:
So In Kotlin Code Cleanup . Should I make this Change too ?